client/packages/system/src/MasterList/Components/MasterListSearchInput.tsx

```diff
@@ -96,7 +104,7 @@ export const MasterListSearchInput = ({
       disabled={disabled}
       width={`${width}px`}
       popperMinWidth={Number(width)}
-      clearable={false}
+      clearable={clearable}
```

Comment:
feedback was raised that the X is more helpful that having to scroll to the bottom of the list for the remove item option, which I agree with. I think when pressed for space, not using the X has some value, but otherwise, we should probably stick with default MUI patterns